code/game/objects/machinery/cloning/cloning.dm

```diff
@@ -241,6 +260,7 @@ You remember nothing of your past life.
 
 You are weak, best rest up and get your strength before fighting.</span>"})
 	occupant.vomit()
-
+	linked_console.radio.talk_into(src, "<b>New clone: [occupant] has been grown in [src] at: [get_area(src)].</b>", RADIO_CHANNEL_MEDICAL)
```

Comment:
maybe some text like for command would be nice here?

Reply:
Command gets the medical channel.

Reply:
i meant for an objective for medical like how command has an objective to give vat to a squad